tests/test_trade.py

```diff
@@ -0,0 +1,154 @@
+"""Testing for the ElfPy package modules"""
```

Comment:
wait I deleted this test & you brought it back

Reply:
merge conflict, removing it.